index.html

```diff
@@ -422,62 +422,62 @@ <h3><dfn>Attributes and default actions</dfn></h3>
                             </thead>
                             <tbody>
                                 <tr>
-                                    <td><a><code>pointerover</code></a></td>
+                                    <td>{{GlobalEventHandlers/pointerover}}</td>
```

Comment:
Very surprising to group an event as "event handler". But if that is the way it is done elsewhere too, fine.

Reply:
more than grouping, it's about scoping/namespacing them - a given event name is guaranteed a reasonable unique interpretation within the interface (or here the mixin) on which it will be emitted.

see https://dontcallmedom.github.io/webdex/d.html#dragstart%40%40GlobalEventHandlers%40event as an example of GlobalEventHandlers used as scope for dragstart.